subsys/config/src/config_fcb.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2015 Runtime Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <fcb.h>
+#include <string.h>
+
+#include "config/config.h"
+#include "config/config_fcb.h"
+#include "config_priv.h"
+
+#define CONF_FCB_VERS		1
+
+struct conf_fcb_load_cb_arg {
+	load_cb cb;
+	void *cb_arg;
+};
+
+static int conf_fcb_load(struct conf_store *, load_cb cb, void *cb_arg);
```

Comment:
struct conf_store *cs? other arguments have names, can follow the same with all arguments.

Reply:
fixed